src/Language/AST/UnionTypeExtensionNode.php

```diff
@@ -12,9 +12,13 @@ class UnionTypeExtensionNode extends Node implements TypeExtensionNode
     /** @var NameNode */
     public $name;
 
-    /** @var DirectiveNode[]|null */
+    /** @var NodeList<DirectiveNode>|null */
     public $directives;
 
-    /** @var NamedTypeNode[]|null */
+    /**
+     * TODO: is this actually a NodeList<NamedTypeNode>?
```

Comment:
It seems like it should be. Have you considered `UnionTypeDefinitionNode`?

Reply:
It seems that the parser writes an array for this one: https://github.com/webonyx/graphql-php/blob/master/src/Language/Parser.php#L1350

Maybe I shouldn't mess with it in this PR, as its fate seems a bit dubious, but it would be great if someone could fix this, or confirm why it is odd duck out.

Reply:
Fixed here: https://github.com/webonyx/graphql-php/pull/694